test/integration/application.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Application', function() {
+  let app;
+  let fooView;
+  let barView;
+
+  beforeEach(function() {
+    cy.server();
+
+    cy.visitMn(function({_, Backbone, Marionette}) {
+      const Application = Marionette.Application.extend({
+        region: '#app-hook',
+      });
+
+      fooView = new Marionette.View({
+        template: _.template('Application region content')
+      });
+      barView = new Marionette.View({
+        template: _.template('<h1 class="title">Application region content</h1>')
+      });
+
+      app = new Application();
+      app.start();
+    });
+  });
+
+  it('should show a view without additional arguments', function() {
+    app.showView(fooView);
```

Comment:
We can do that.. but in some instances it may not be possible because the library instances are only available to that function and we shouldn't let them bleed outside I don't think.  But yeah it seems super reasonable in this case.. otherwise we'd have soooo many `visitMn`s